influxdb3/src/commands/serve.rs

```diff
@@ -345,6 +345,13 @@ pub struct Config {
         action
     )]
     pub telemetry_endpoint: String,
+
+    /// Set the limit for number of parquet files allowed in a query. Defaults to 432 which is about
+    /// 3 days worth of files. This number can be increased to allow more files to be queried, but
+    /// query performance will likely suffer as a result. It would be better to specify smaller time
```

Comment:
Worth noting that it's not just that query performance may suffer, but the any query going over this limit will spike RAM usage significantly and depending on system resources could get OOMkilled